api/tests/unit/domain/models/PrivateCertificate_test.js

```diff
@@ -0,0 +1,78 @@
+const { expect, domainBuilder } = require('../../../test-helper');
+const PrivateCertificate = require('../../../../lib/domain/models/PrivateCertificate');
+const { status: assessmentResultStatuses } = require('../../../../lib/domain/models/AssessmentResult');
+
+describe('Unit | Domain | Models | PrivateCertificate', () => {
+
+  context('#static buildFrom', () => {
+
+    const commonData = {
+      id: 1,
+      firstName: 'Jean',
+      lastName: 'Bon',
+      birthdate: '2000-03-20',
+      birthplace: 'Sarajevo',
+      isPublished: true,
+      userId: 123,
+      date: '2019-01-01',
+      deliveredAt: new Date('2019-05-05'),
+      certificationCenter: 'Centre des fruits et légumes',
+      pixScore: 250,
+      commentForCandidate: 'Bravo !',
+      cleaCertificationStatus: 'someCleaStatus',
+      resultCompetenceTree: null,
+      verificationCode: 'someVerifCode',
+      maxReachableLevelOnCertificationDate: 5,
+    };
+
+    it('builds a PrivateCertificate cancelled', async () => {
```

Comment:
Description de test : ici et dans les descriptions suivantes, je propose d'inverser le nom et l'adjectif, par exemple `it('builds a cancelled PrivateCertificate', ...)`.